multi-node-k8s.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+# TODO: disable debugging
+set -x
```

Comment:
you can also add `set -e -o pipefail` and make sure to exit the script as soon as a command returned an error

Reply:
The `set -e` is something that you can keep, no command is expected to fail in this script.